blog/content/post/2021-02-23-OSF_vPub_01.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "OSF vPub (vBeer) - what a fun event!"
+abstract: "We've just had our first OSF vPub gathering, with lots of
+          fun & really interesting talk about the open-source
+          firmware/hardware with 50 masters from all over the world!"
+cover: /covers/OSF_vPub.png
+author: mike.banon
+layout: post
+published: true
+date: 2021-02-23
+archives: "2021"
+
+tags:
+  - Conference
+  - Event
+  - vPub
+  - Firmware
+  - Open-source
+categories:
+  - Firmware
+  - Miscellaneous
+
+---
+
+Our meeting has been attended by ~50 firmware masters from all over the world!
+
+Together with our prominent 3mdeb hosts - Piotr Król and Michał Żygowski - it
+was an honour to be visited by such rockstars as Thierry Laurion from [Insurgo][1],
+Tom Hiller aka Thrilleratplay, Daniel Maslowski aka CyReVolt, Piotr Kubaj,
+Andrew Cooper, Felix Singer, David Hendricks, and many others whom I forgot to
+mention (sorry!) - or who preferred to stay anonymous and participate via chat.
+
+We've also had some surprise guests: despite being the relative newcomers to our
+cosy community, they have managed to ignite quite an interesting discussion!
+Alexey Vazhnov described how tricky could it be to get merged the coreboot docs,
+and we've thought about how to improve this workflow & make this process easier.
```

Comment:
@miczyg1 did we managed to do something with that? @mikebdp2 can we link discussion from reddit?

Reply:
@pietrushnic Done